src/content/docs/guides/configure-biome.mdx

```diff
@@ -213,7 +213,7 @@ In the following example, we tell Biome to include files, excluded the `dist` di
   "files": {
     "includes": [
       "**",
-      "!**/dist/**",
```

Comment:
I realised that what we say at line  `209` isn't entirely correct. `!**/dist` esclused **any** `dist` directory. It might be a nitpick, should we update it? Or maybe we should update the glob to `!dist`